lib/rubocop/plugin/context_creator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'lint_roller/context'
+
+module RuboCop
+  module Plugin
+    # A class for creating plugin resolution contexts within the plugin runner.
+    # @api private
+    class ContextCreator
```

Comment:
Do we really need a dedicated class just for this? Why not have it at the module level?

Reply:
This has been inlined into `RuboCop::Plugin::ConfigurationIntegrator`.